libs/lb/lb-rs/Cargo.toml

```diff
@@ -24,32 +24,34 @@ bezier-rs = "0.2.0"
 bincode = "1.3.3"
 time = "0.3.20"
 diffy = "0.3.0"
+db-rs = {git="https://github.com/lockbook/db-rs"}
```

Comment:
is this intended to stay this way? also, nit: formatting

Reply:
not intended, the change has been released